README.md

```diff
@@ -186,6 +187,33 @@ In other words, if the endpoint is a _qcow2_ file, or any file wrapped by tar (e
 
 An additional caveat is that currently the size is under reported for simple iso files, meaning a structured iso file that is not archived and/or compressed.
 
+### Start Cloning Images
```

Comment:
This seems out of place in the doc. It should be before _Endpoint Size_ (which also seems a bit out of place, I was the author of that section). There should be a work-flow order to the _Deploy the API Objects_ section.

Reply:
moved.